lib/Signaling/Messages.php

```diff
@@ -112,7 +112,7 @@ public function addMessageForAllParticipants(Room $room, string $message): void
 	 * "new" message.
 	 *
 	 * @param string $sessionId
-	 * @return array
+	 * @return array{type: string, data: string}[]
```

Comment:
```suggestion
	 * @return list<array{type: string, data: string}>
```

Reply:
This doesn't work because psalm will complain about more/less specific return types. `array<...>` works though

Reply:
Psalm was just not able to detect it, but it actually always was a list. Anyway, pushed a small commit that fixes it.

Reply:
feel free to squash all together now and rebuild openapi